core/services/bulletprooftxmanager/eth_confirmer_test.go

```diff
@@ -2355,10 +2331,11 @@ func TestEthConfirmer_ForceRebroadcast(t *testing.T) {
 func TestEthConfirmer_ResumePendingRuns(t *testing.T) {
 	t.Parallel()
 
-	db := pgtest.NewGormDB(t)
-	sqlxdb := postgres.UnwrapGormDB(db)
+	db := pgtest.NewSqlxDB(t)
+	gdb := pgtest.GormDBFromSql(t, db.DB)
+	borm := cltest.NewBulletproofTxManagerORM(t, db)
 
-	ethKeyStore := cltest.NewKeyStore(t, sqlxdb).Eth()
+	ethKeyStore := cltest.NewKeyStore(t, db).Eth()
```

Comment:
We seem to be doing this in every test. Would be useful to have a test setup function. Not a blocker for approval here but nice to have.